### ISSUE TITLE
UnaryInterceptor no getResponseMessage method

### ISSUE THREAD

**Issue by jackielii** [grpc/grpc-web#918]
Hi there,

I have the following UnaryInterceptor:

```ts
class UnaryAuthInterceptor<REQ extends Request, RESP extends UnaryResponse> {
  async intercept(request: REQ, invoker: (request: REQ) => Promise<RESP>) {
    const md = request.getMetadata()
    md['Authorization'] = `Bearer ${getAuthToken()}`
    if (process.env.NODE_ENV === 'development') {
      console.log('grpc-web request:', request.getRequestMessage()?.toObject(), 'metadata:', md)
    }
    // cancellation
    // const signal = md.signal // UnaryCall doesn't allow cancellation
    delete md.signal
    try {
      const resp = await invoker(request)
      if (process.env.NODE_ENV === 'development') {
        console.log('grpc-web unary response:', resp.getResponseMessage()) // <<<==== fails here
      }
      return resp
    } catch (e) {
      if (process.env.NODE_ENV === 'development') {
        console.log('grpc-web unary error', e)
      }
      throw e
    }
  }
}
```

`Response.getResponseMessage` appear to be exported but actually it's not available

**Comment by stanley-cheung** [comment]
You are probably running into the same issue as [this](https://github.com/grpc/grpc-web/issues/909#issuecomment-662013369). This should be fixed in `master` and we should be cutting a new release soon.

**Comment by stanley-cheung** [comment]
This should be fixed in `1.2.1`.
